corehq/apps/accounting/invoicing.py

```diff
@@ -849,6 +849,7 @@ def _end_date_count_sms(self):
     @property
     @memoized
     def unit_cost(self):
+        #  This seems to return aggregate cost of all the excess SMS, not actually "unit cost"
```

Comment:
It looks like the purpose of this method is part of implementing `FeatureLineItemFactory`. What do you think of rewording this and turning it into a doc string?
```suggestion
        """Return aggregate cost of all the excess SMS"""
```